src/client/src/ui/footer/model/footerModel.js

```diff
@@ -3,10 +3,18 @@ import { CompositeStatusService } from '../../../services';
 import { logger } from '../../../system';
 import { ModelBase } from '../../common';
 import { ServiceStatusLookup } from '../../../services/model';
+import _ from 'lodash';
 
 var _log:logger.Logger = logger.create('FooterModel');
 
-export default class FooterModel extends ModelBase {
+export const FooterConnectionStatus = {
```

Comment:
can we put this in a separate file and follow the same pattern we've been using for other status? for exmaple [tradeStatus](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/dev/src/client/src/services/model/tradeStatus.js)